hyppo/ksample/energy.py

```diff
@@ -106,16 +107,18 @@ def statistic(self, x, y):
         stat : float
             The computed Energy statistic.
         """
-        distx = x
-        disty = y
+        dataMatrix = np.concatenate([x, y])
+        labelMatrix = np.concatenate([np.zeros(len(x)), np.ones(len(y))])
```

Comment:
This two lines are not necessary as specified in Nonpar Manova via Independence testing paper. Dcov and Energy are equivalent up to a transformation. So, it is more efficient to compute the dcorr statistic and transform it